package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "webpack": "webpack --config webpack.config.js --watch",
-    "webpack_production": "webpack --config webpack_production.config.js"
+    "webpack_production": "webpack --config webpack_production.config.js",
+    "merge-regions": "mapshaper -i source/seedsource/static/sst/geometry/*.json combine-files -o source/seedsource/static/sst/geometry/regions.topojson"
```

Comment:
Nice! I like having this as a build step so we don't have to worry about converting to topojson separately, or merging into a single file.